pkg/ls/labels.go

```diff
@@ -0,0 +1,17 @@
+package ls
+
+import (
+	"github.com/cloudprivacylabs/lpg"
+)
+
+func ProcessLabeledAs(graph *lpg.Graph) {
+	for nodeItr := graph.GetNodes(); nodeItr.Next(); {
+		node := nodeItr.Node()
+		labels := node.GetLabels()
+		if AsPropertyValue(node.GetProperty(AttributeNameTerm)).AsString() == "labeledAs" {
```

Comment:
No, we are not looking for attributeName=labeledAs. We are looking for ls:labeledAs as property key.
So, AsPropertyValue(node.GetProperty).MustStringSlice() is the properties we need.